lib/vsc/install/shared_setup.py

```diff
@@ -291,7 +291,7 @@ def get_egg_cache_dir_pyver(self):
 
         # the original get_egg_cache_dir creates the .eggs directory if it doesn't exist yet,
         # but we want to have it versioned, so we rename it
-        egg_cache_dir_pyver = egg_cache_dir + '.py%s' % sys.version_info[0]
+        egg_cache_dir_pyver = egg_cache_dir + '.py%s' % '.'.join([str(x) for x in sys.version_info[0:2]])
```

Comment:
is this strictly needed?

Reply:
Maybe not but it's a lot cleaner not to mix all python3 versions.

I guess it's not needed if you have a clean directory every time.